docs/src/manual/boundary_conditions.md

```diff
@@ -280,8 +280,8 @@ A Differential Algebraic Equations (DAE) is an equation of the form
 which usually cannot be expressed as a true ODE. They occur often,
 but not always, in forms where some time derivatives are missing
 ```math
-u_1'(t) = f(\boldsymbol{u}(t),t)
-0 = g(\boldsymbol{u}(t),t)`
+u_1'(t) = f(\boldsymbol{u}(t),t)\\
+0 = g(\boldsymbol{u}(t),t)
```

Comment:
```suggestion
0 = g(\boldsymbol{u}(t),\boldsymbol'{u}(t),t)
```
Do you mean something like this? 
Feel free to edit the branch as you see fit - this is really your area of expertise!

Reply:
I will address this in a separate PR as I need to think a bit about how to formulate the issue in an understandable way.

Reply:
OK, in that case, I think it is better to remove this temporarily as no docs are better than wrong docs.